DiffusionPreprocessing/DiffPreprocPipeline.sh

```diff
@@ -513,12 +512,12 @@ main() {
 	pre_eddy_cmd+=" --echospacing=${echospacing} "
 	pre_eddy_cmd+=" --b0maxbval=${b0maxbval} "
 	pre_eddy_cmd+=" --printcom=${runcmd} "
-	if [ ! -z "${SelectBestB0}" ]; then
+	if [ "${SelectBestB0}" == "true" ]; then
 		pre_eddy_cmd+=" --select-best-b0 "
 	fi
 
 	log_Msg "pre_eddy_cmd: ${pre_eddy_cmd}"
-	${pre_eddy_cmd}
+	#${pre_eddy_cmd}
```

Comment:
I'm guessing these were commented out for testing the variable printing, and weren't intended to be committed this way?  See also lines 545 and 562.

Reply:
Oops